src/release_windows.go

```diff
@@ -152,4 +152,17 @@ func cleanUp() {
 	if verboseEnabled {
 		warnPrint("Done with automatic cleanup! You need to remove aeacus.exe manually. The only things you need in the C:\\aeacus directory is phocus, scoring.dat, TeamID.txt, and the assets directory.")
 	}
+	if verboseEnabled {
+		warnPrint("clearing run history")
+	}
```

Comment:
do we still wanna do verboseEnabled here?

Reply:
It should be until #58 is merged

Reply:
oh your right

Reply:
well shanes thing hasnt been integrated into aeacus yet, i can wait for shanes pr to be merged and then change mine